Hunting Queries/TeamsLogs/ExternalUserAddedRemoved.yaml

```diff
@@ -0,0 +1,30 @@
+id: e9cef368-6257-4dab-9a48-4793729c46a2
+name: External user added and removed in short timeframe
+description: |
+  'This hunting query identifies external user accounts that are added to a Team and then removed within
+  one hour.
+  This query requires you to have Teams data collected from the O365 Management Activity API
+  and the Teams parser function enabled.'
+tactics:
+  - Persistance
+relevantTechniques:
+  - T1136
+query: |
+
+  // If you want to look at user added further than 7 days ago adjust this value
+  let time_ago = 7d;
+  // If you want to change the timeframe of how quickly accounts need to be added and removed change this value
+  let time_delta = 1h;
+  TeamsData 
+  | where TimeGenerated > ago(time_ago)
+  | where Operation == "MemberAdded"
+  | extend UPN = tostring(parse_json(Members)[0].UPN)
```

Comment:
We should bring thru TeamName and Details.TeamGuid value as this helps identify the specific Team.

Reply:
Done, and extracted TeamGuid in a few other places as well for clarity.